autoswitch_virtualenv.plugin.zsh

```diff
@@ -38,10 +38,18 @@ function _check_venv()
 
         SWITCH_TO=""
 
+        # Get the .venv file, scanning parent directories
         venv_path=$(_check_venv_path "$PWD")
         if [[ -n "$venv_path" ]]; then
-          file_owner="$(stat -c %u "$venv_path")"
-          file_permissions="$(stat -c %a "$venv_path")"
+
+          stat --version &> /dev/null
```

Comment:
looks like the only other thing would be to parse the `ls` output which seems like a worse alternative